x-pack/test/apm_api_integration/tests/service_groups/service_groups_api_methods.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ApmApiClient } from '../../common/config';
+
+export async function getServiceGroupsApi(apmApiClient: ApmApiClient) {
+  return apmApiClient.writeUser({
+    endpoint: 'GET /internal/apm/service-groups',
+  });
+}
+
+export async function createServiceGroupApi({
+  apmApiClient,
+  serviceGroupId,
+  groupName,
+  kuery,
+  description,
+  color,
+}: {
+  apmApiClient: ApmApiClient;
+  serviceGroupId?: string;
+  groupName: string;
+  kuery: string;
+  description?: string;
+  color?: string;
+}) {
+  const response = await apmApiClient.writeUser({
+    endpoint: 'POST /internal/apm/service-group',
+    params: {
+      query: {
+        serviceGroupId,
+      },
+      body: {
+        groupName,
+        kuery,
+        description,
+        color,
+      },
```

Comment:
This would then be:

```suggestion
      body: serviceGroup,
```

Reply:
I'm getting this 
<img width="1659" alt="image" src="https://user-images.githubusercontent.com/1313018/213164141-5d7a9f36-b842-4bb5-b353-920caa2c2cbf.png">

The only way to reuse the type is doing something like this
<img width="876" alt="image" src="https://user-images.githubusercontent.com/1313018/213164534-6340a32b-bd8a-40f0-af14-e775ae8dbb20.png">

Should be still use it?